internal/printer/select.go

```diff
@@ -23,13 +23,16 @@ import (
 	"fmt"
 	"github.com/lithammer/fuzzysearch/fuzzy"
 	"github.com/pterm/pterm"
+	"github.com/version-fox/vfox/internal"
 	"github.com/version-fox/vfox/internal/logger"
+	"github.com/version-fox/vfox/internal/util"
 	"sort"
 	"strings"
 )
 
 type PageKVSelect struct {
 	index             int
+	InstallVersions   util.Set[internal.Version]
```

Comment:
I think `HighlightOptions` name may be more general than `InstallVersions`. 

For types, `string` is fine.